firmware/V3.x/RP2040/LoRa-CLI/LoRa-CLI.ino

```diff
@@ -91,9 +92,9 @@ void setup() {
 
   // initialize SX1262 with default settings
   Serial.print(F("[SX1262] Initializing ... "));
-  //debing(float freq, float bw, uint8_t sf, uint8_t cr, uint8_t syncWord, int8_t power, uint16_t preambleLength, float tcxoVoltage, bool useRegulatorLDO)
 
-  int state = radio.begin(915.0, 250, 7, 5, 0x12, 20, 8, 0, false);
+  //(float freq, float bw, uint8_t sf, uint8_t cr, uint8_t syncWord, int8_t power, uint16_t preambleLength, float tcxoVoltage, bool useRegulatorLDO)
+  int state = radio.begin(915.0, 125.0, 7, 5, 0x12, 20, 8, 0, false);
```

Comment:
Same here

Reply:
It was missing to show the default parameters (which are also used as default by several LoRa libraries in their basic examples) to the user so that the user has the reference before changing them in the CLI. The reason why they are as magic numbers is that the bw is referenced inside the set_bw() function, so I preferred to do it this way for the moment. I will see the possibility of removing the reference (switch/cases) and then and then variables could be used as you suggest.

Reply:
Ok, no problem